src/unix/bsd/freebsdlike/freebsd/freebsd11/mod.rs

```diff
@@ -1,488 +0,0 @@
-// APIs that were changed after FreeBSD 11
-
-// The type of `nlink_t` changed from `u16` to `u64` in FreeBSD 12:
-pub type nlink_t = u16;
```

Comment:
All of the stuff that was in this file should now be moved into freebsdlike/freebsd/mod.rs  For example, move all three `nlink_t` definitions from freebsd{12,13,14}.rs into that file.